### ISSUE TITLE
Please consider updating the Chromium version used by the client.

### ISSUE THREAD

**Issue by francescotissera1211** [ValveSoftware/steam-for-linux#10077]
As a blind user of Steam on Linux, I am writing to express my disappointment at the lack of accessibility features in the client. The current version of Chromium used in Steam is, I presume, outdated, which prevents blind users from using screen readers to navigate the interface.

This is a major barrier to accessibility, and it prevents blind users from being able to use Steam independently. We have the right to use and enjoy Steam just as much as any other group of users, but we are unable to do so without sighted assistance.

I urge you to prioritize accessibility and update the version of Chromium used in Steam on Linux. This would allow blind users to use screen readers to navigate the interface and use Steam independently.

I understand that you are busy, but I believe that this is an important issue that needs to be addressed. Thank you for your time and consideration.

Please note that I have not put the information requested before the description because firstly it does not matter when it comes to this issue and secondly because I cannot  get the steam client version and build number, since the info page, if one exists, is not accessible.

**Comment by kisak-valve** [comment]
Hello @francescotissera1211, looking this issue report over, I do not believe that your previous issue report at #9911 has additional complications due to the version of libcef embedded in Steam. We'll need to hear from a Steam dev for any progress related to that issue report.

Closing in favor of the older issue report.
